.github/workflows/examples.yml

```diff
@@ -16,21 +20,11 @@ jobs:
 
       - name: Install the project and its dependencies
         run: |
-          uv sync
-          uv sync --extra ingress-torch-cpu
-
-      - name: Run MLP From File
-        run: |-
-          uv run python/examples/ingress/torch/mlp_from_file.py
+          sudo apt-get install llvm-dev # Obtain FileCheck, used in testing.
```

Comment:
Use `-y` to make sure it doesn't lock asking `[y/N]`

Reply:
Done.